.github/workflows/nightly.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   linux:
     name: Linux
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
We should use ubuntu-20.04 with proper `apt-get` fix

Reply:
yeah, as a consumer of the nightlies, I would prefer that too, if possible

Reply:
+1 agreed

Reply:
@AkihiroSuda need do some tests on it? 

Basically, we need add `btrfs-progs-devel` to the apt install list. 

Let me try.

Reply:
```
E: Unable to locate package btrfs-progs-devel:amd64
```

Reply:
tested with 
```
            libbtrfs-dev:amd64 \
            libbtrfs-dev:arm64 \
            libbtrfs-dev:s390x \
            libbtrfs-dev:ppc64el
``` 
and it works for me

Reply:
wait for testing result in https://github.com/pacoxu/containerd/runs/2164779752?check_suite_focus=true

Reply:
@aojea 
It works. 👍